samples/basic-approval-workflow/src/app.js

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Paragraph, Note } from '@contentful/forma-36-react-components';
+import extensionDescriptor from '../extension.json';
+
+import { eventTypes } from './events';
+import id from './id';
+import findUserById from './find-user';
+import userName from './user-name';
+
+import Vspace from './vspace';
+import Log from './log';
+import ReviewBox from './review-box';
+
+export default class App extends React.Component {
+  state = {}
+
+  static propTypes = {
+    log: PropTypes.arrayOf(PropTypes.object).isRequired,
+    users: PropTypes.arrayOf(PropTypes.object).isRequired,
+    sdk: PropTypes.shape({
+      entry: PropTypes.shape({
+        onSysChanged: PropTypes.func.isRequired,
+        getSys: PropTypes.func.isRequired,
+      }).isRequired,
+      space: PropTypes.shape({
+        getEntry: PropTypes.func.isRequired,
+        publishEntry: PropTypes.func.isRequired,
+      }),
+      user: PropTypes.object.isRequired,
+      dialogs: PropTypes.shape({
+        openExtension: PropTypes.func.isRequired,
+      }).isRequired,
+      notifier: PropTypes.shape({
+        success: PropTypes.func.isRequired,
+        error: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
+    pubnub: PropTypes.shape({
+      publish: PropTypes.func.isRequired,
+    }).isRequired,
+  }
+
+  componentDidMount() {
+    const { sdk } = this.props;
+    this.off = sdk.entry.onSysChanged((sys) => {
+      this.setState({ currentVersion: sys.version });
+    });
+  }
+
+  componentWillUnmount() {
+    if (typeof this.off === 'function') {
+      this.off();
+    }
+  }
+
+  render = () => {
+    const { log, users } = this.props;
+    const { currentVersion, busy } = this.state;
+
+    const last = log[0];
```

Comment:
As we're destructuring everywhere above:

```suggestion
    const [last] = log;
```